.vscode/scripts/import-example-app.sh

```diff
@@ -30,8 +30,18 @@ else
   pip-compile -r -q ./requirements.in
   pip-sync ./requirements.txt ./app/requirements.txt ./app/tests/requirements.txt
 
+  # Workaround to restore dependency on velocitas-sdk and generate new vehicle model
+  sudo rm -rf ~/.velocitas
+  velocitas init
+  velocitas sync
```

Comment:
IMO this is unnecessary overhead since the velocitas.json is not changing...

Reply:
Yes, therefore it is only a workaround. Unfortunately, `velocitas exec sdk-installer run` is failing as an independent command. I will investigate it, but for now, we need a solution to unblock some parties.